packages/cdktf-cli/templates/java/src/main/java/com/mycompany/app/Main.java

```diff
@@ -5,17 +5,13 @@
 import com.hashicorp.cdktf.App;
 import com.hashicorp.cdktf.TerraformStack;
 
-public class Main extends TerraformStack
-{
-    public Main(final Construct scope, final String id) {
-        super(scope, id);
-
-        // define resources here
-    }
+import com.mycompany.app.MainStack;
```

Comment:
This should be in the `com.mycompany.app` package and then this import won't be necessary.